common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -135,6 +137,7 @@ protected ArmClient GetArmClient(ArmClientOptions clientOptions = default, strin
             options.AddPolicy(ResourceGroupCleanupPolicy, HttpPipelinePosition.PerCall);
             options.AddPolicy(ManagementGroupCleanupPolicy, HttpPipelinePosition.PerCall);
             options.AddPolicy(NullFilterPolicy, HttpPipelinePosition.PerRetry);
+            options.AddPolicy(LROCheck, HttpPipelinePosition.PerCall);
```

Comment:
well if this is only a test, this is fine.
But we should not add policies like this to our test base.